MangoPay/UboDeclaration.php

```diff
@@ -19,6 +20,12 @@ class UboDeclaration extends Libraries\EntityBase
      */
     public $Status;
 
+    /**
+     * The output status of Declaration status.
+     * @var string
+     */
+    public $OutputStatus;
```

Comment:
Why did you add this param? I don't see it in the docs (correct me if I'm wrong)

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.